src/styles/blocks/card.scss

```diff
@@ -0,0 +1,23 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background: $white-color;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+  perspective: 100px;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 39px 0 45px;
+    transition: transform 400ms;
+
+    &:hover {
+      transform: translate3d(0, 0, 20px);
```

Comment:
Use simple transform: translate here, you picture now jumps on hover. It shouldn't.